docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -17,7 +17,7 @@ NuGet's behavior is driven by the accumulated settings in one or more `NuGet.Con
 | --- | --- | --- |
 | Solution | Current folder (aka Solution folder) or any folder up to the drive root.| In a solution folder, settings apply to all projects in subfolders. Note that if a config file is placed in a project folder, it has no effect on that project. When restoring a project on the command line, the project's directory is treated as the solution directory, which can lead to differences in behaviour when restoring the project vs solution. |
 | User | **Windows:** `%appdata%\NuGet\NuGet.Config`<br/>**Mac/Linux:** `~/.config/NuGet/NuGet.Config` or `~/.nuget/NuGet/NuGet.Config` (varies by tooling) <br/>Additional configs are supported on all platforms. These configs cannot be edited by the tooling. </br> **Windows:** `%appdata%\NuGet\config\*.Config` <br/>**Mac/Linux:** `~/.config/NuGet/config/*.config` or `~/.nuget/config/*.config` | Settings apply to all operations, but are overridden by any solution-level settings. |
-| Computer | **Windows:** `%ProgramFiles(x86)%\NuGet\Config`<br/>**Mac/Linux:** `$XDG_DATA_HOME`. If `$XDG_DATA_HOME` is null or empty, `~/.local/share` or `/usr/local/share` will be used (varies by OS distribution)  | Settings apply to all operations on the computer, but are overridden by any user- or solution-level settings. |
+| Computer | **Windows:** `%ProgramFiles(x86)%\NuGet\Config`<br/>**Mac/Linux:** `$XDG_DATA_HOME`. /etc/opt/NuGet/Config by default. If $NUGET_COMMON_APPLICATION_DATA is neither null nor empty, then $NUGET_COMMON_APPLICATION_DATA/NuGet/Config instead  | Settings apply to all operations on the computer, but are overridden by any user- or solution-level settings. |
```

Comment:
Isn't any mention of `XDG_DATA_HOME` incorrect?  The way i read the commentary and the source its not mentioned at all in the context of locating the `NuGet.Config` file.  So shouldn't this be:

```suggestion
| Computer | **Windows:** `%ProgramFiles(x86)%\NuGet\Config`<br/>**Mac/Linux:** `/etc/opt/NuGet/Config` (Linux) or `/Library/Application Support` (Mac) by default. If `$NUGET_COMMON_APPLICATION_DATA` is neither null nor empty, then `$NUGET_COMMON_APPLICATION_DATA/NuGet/Config` instead  | Settings apply to all operations on the computer, but are overridden by any user- or solution-level settings. |
```

Reply:
@heng-liu, please, rephrase as you deem fit. I am not 100% sure which aspects of proposed changes you found inaccurate or incomplete.